systemd/nextcloud-desktop.service.in

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2021 Andrea Pappacoda <andrea@pappacoda.it>
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+[Unit]
+Description=Nextcloud desktop user service
+Wants=network-online.target
+
+[Service]
+ExecStart=@bindir@/nextcloud --background
```

Comment:
executable name is defined as a variable in cmake files
please replace this by this variable `APPLICATION_EXECUTABLE`